platform/landing-page/src/main/resources/index.handlebars

```diff
@@ -164,7 +164,7 @@
     <div class="container">
         <div class="row">
             <div class="col-md-6">
-                <h2>Data Source Availability</h2>
+                <h2>{{sourceKeyword}} Availability</h2>
```

Comment:
✏️ Usually, you would want the whole sentence to be internationalized and not just parts of it since in some language later, words might be reversed and such.

Reply:
I agree with making this change. Originally when I was doing keyword replacement I think it made sense to only replace the parts of the phrases that were configurable but now that we're doing internationalization it makes more sense to replace the entire phrase.